stdimage/management/commands/rendervariations.py

```diff
@@ -77,9 +89,16 @@ def render(field, images, count, replace, do_render):
 
 def render_field_variations(kwargs):
     kwargs['storage'] = get_storage_class(kwargs['storage'])()
+    igrnore_missing = kwargs.pop('igrnore_missing')
     do_render = kwargs.pop('do_render')
-    if callable(do_render):
-        kwargs.pop('field_class')
-        do_render = do_render(**kwargs)
-    if do_render:
-        render_variations(**kwargs)
+    try:
+        if callable(do_render):
+            kwargs.pop('field_class')
+            do_render = do_render(**kwargs)
+        if do_render:
+            render_variations(**kwargs)
+    except FileNotFoundError as e:
+        if not igrnore_missing:
+            raise
```

Comment:
I would recommend to cast to a `CommandError` and provide a more descriptive error message, eg:
```python
raise CommandError("Original file was not found, terminating command. Use --ignore-missing to skip this error.") from e
```